Tests/SwiftDocCTests/XCTestCase+LoadingTestData.swift

```diff
@@ -0,0 +1,79 @@
+        context.externalReferenceResolvers = externalResolvers
+        context.externalSymbolResolver = externalSymbolResolver
+        context.externalMetadata.diagnosticLevel = diagnosticFilterLevel
+        try configureContext?(context)
+        // Load the bundle using automatic discovery
+        let automaticDataProvider = try LocalFileSystemDataProvider(rootURL: bundleURL)
+        // Mutate the bundle to include the code listings, then apply to the workspace using a manual provider.
+        var bundle = try automaticDataProvider.bundles().first!
+        bundle.attributedCodeListings = codeListings
+        let dataProvider = PrebuiltLocalFileSystemDataProvider(bundles: [bundle])
+        try workspace.registerProvider(dataProvider)
+        return (bundleURL, bundle, context)
+    }
+    
+    func testBundleAndContext(copying name: String, excludingPaths excludedPaths: [String] = [], codeListings: [String : AttributedCodeListing] = [:], externalResolvers: [BundleIdentifier : ExternalReferenceResolver] = [:], externalSymbolResolver: ExternalSymbolResolver? = nil,  configureBundle: ((URL) throws -> Void)? = nil) throws -> (URL, DocumentationBundle, DocumentationContext) {
+        let sourceURL = Bundle.module.url(
+            forResource: name, withExtension: "docc", subdirectory: "Test Bundles")!
+        
+        let sourceExists = FileManager.default.fileExists(atPath: sourceURL.path)
+        let bundleURL = sourceExists
+            ? try createTemporaryDirectory().appendingPathComponent("\(name).docc")
+            : try createTemporaryDirectory(pathComponents: "\(name).docc")
+        
+        if sourceExists {
+            try FileManager.default.copyItem(at: sourceURL, to: bundleURL)
+        }
+        
+        for path in excludedPaths {
+            try FileManager.default.removeItem(at: bundleURL.appendingPathComponent(path))
+        }
+        
+        // Do any additional setup to the custom bundle - adding, modifying files, etc
+        try configureBundle?(bundleURL)
+        
+        return try loadBundle(from: bundleURL, codeListings: codeListings, externalResolvers: externalResolvers, externalSymbolResolver: externalSymbolResolver)
+    }
+    
+    func testBundleAndContext(named name: String, codeListings: [String : AttributedCodeListing] = [:], externalResolvers: [String: ExternalReferenceResolver] = [:]) throws -> (DocumentationBundle, DocumentationContext) {
+        let bundleURL = Bundle.module.url(
+            forResource: name, withExtension: "docc", subdirectory: "Test Bundles")!
+        let (_, bundle, context) = try loadBundle(from: bundleURL, codeListings: codeListings, externalResolvers: externalResolvers)
+        return (bundle, context)
+    }
+    
+    func testBundle(named name: String) -> DocumentationBundle {
+        let (bundle, _) = try! testBundleAndContext(named: name)
```

Comment:
```suggestion
    func testBundle(named name: String) throws -> DocumentationBundle {
        let (bundle, _) = try testBundleAndContext(named: name)
```

Reply:
I like this but I'll need to make a separate commit to also update all the call sites to use `try`